examples/labelanchor-example.ts

```diff
@@ -166,39 +169,62 @@ class Position2DLabelRenderer extends Renderer {
-        // among the stalks, and grow familiar with the countless indescribable forms of the insects and flies, then I feel the \
-        // presence of the Almighty, who formed us in his own image, and the breath of that universal love which bears and \
-        // sustains us, as it floats around us in an eternity of bliss;  and then, my friend, when darkness overspreads my eyes, \
-        // and heaven and earth seem to dwell in my soul and absorb its power, like the form of a beloved mistress, then I often \
-        // think with longing, Oh, would I could describe these conceptions, could impress upon paper all that is living so full \
-        // and warm within me, that it might be the mirror of my soul, as my soul is the mirror of the infinite God!';
-        // tslint:enable:max-line-length
-
-        const label = new Position2DLabel(new Text(`ABVAg'^|\nAAVAVAVA`), Label.Type.Dynamic);
-        label.fontSize = 128;
-        label.fontSizeUnit = Label.Unit.Pixel;
-        label.alignment = Label.Alignment.Left;
-        label.lineAnchor = Label.LineAnchor.Center;
-        label.position = [-555.0, +0.0];
-
-        this._labelPass.labels = [label];
+        const kafka = 'One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in \
+his bed into a horrible vermin. He lay on his armour-like back, and if he lifted his head a little he could see his \
+brown belly, slightly domed and divided by arches into stiff sections.';
+
+        this._labelTop = new Position2DLabel(new Text(`Label.Anchor.Top |  ${kafka}`), Label.Type.Dynamic);
+        this._labelTop.lineAnchor = Label.LineAnchor.Top;
+
+        this._labelAscent = new Position2DLabel(new Text(`Label.Anchor.Ascent |  ${kafka}`), Label.Type.Dynamic);
+        this._labelAscent.lineAnchor = Label.LineAnchor.Ascent;
+
+        this._labelCenter = new Position2DLabel(new Text(`Label.Anchor.Center |  ${kafka}`), Label.Type.Dynamic);
+        this._labelCenter.lineAnchor = Label.LineAnchor.Center;
+
+        this._labelBaseline = new Position2DLabel(new Text(`Label.Anchor.Baseline |  ${kafka}`), Label.Type.Dynamic);
+        this._labelBaseline.lineAnchor = Label.LineAnchor.Baseline;
+
+        this._labelDescent = new Position2DLabel(new Text(`Label.Anchor.Descent |  ${kafka}`), Label.Type.Dynamic);
+        this._labelDescent.lineAnchor = Label.LineAnchor.Descent;
+
+        this._labelBottom = new Position2DLabel(new Text(`Label.Anchor.Bottom |  ${kafka}`), Label.Type.Dynamic);
+        this._labelBottom.lineAnchor = Label.LineAnchor.Bottom;
+
+        this._labelPass.labels = [this._labelTop, this._labelAscent, this._labelCenter,
+        this._labelBaseline, this._labelDescent, this._labelBottom];
+
+        for (const label of this._labelPass.labels) {
+            label.fontSize = 20;
+            label.color.fromHex('#fff');
+            label.fontSizeUnit = Label.Unit.Pixel;
```

Comment:
This line has no actual effect at the current implementation, right?
Is this for future features, like `Label.Uni.Pointt?`